### PR TITLE
sydent: Add support for matrix_is_tester

### PR DESCRIPTION
**Pull request by dbkr**
<del>nb. This is pinned to a different branch and should be updated to point to master once https://github.com/matrix-org/matrix-is-tester/pull/5/ is merged.</del> Now updated

### CODE REVIEW

File: sydent/pipeline.yml [hunk 1]
```diff
@@ -5,3 +5,18 @@ steps:
     plugins:
       - docker#v3.0.1:
           image: "matrixdotorg/sydent-build"
+  - command:
+      - "echo '--- Install dependencies'"
+      - "pip install -e ."
+      - "echo '--- Install matrix-is-tester'"
+      - "python -m pip install git+https://github.com/matrix-org/matrix-is-tester.git"
```
Comment (richvdh) [inline sydent/pipeline.yml]: why do we `pip install` above and `python -m pip install` here?

File: sydent/pipeline.yml [hunk 2]
```diff
@@ -5,3 +5,18 @@ steps:
...
+      - "echo '+++ Run Tests'"
+      - "PYTHONPATH=`pwd` trial matrix-is-tester"
```
Comment (richvdh) [inline sydent/pipeline.yml]: what's the point of setting PYTHONPATH if we've already pip installed it?

**Review by richvdh — COMMENTED** [review]
Couple of questions.

> nb. This is pinned to a different branch and should be updated to point to master once matrix-org/matrix-is-tester#5 is merged.

this is outdated?

File: sydent/pipeline.yml (hunk 1 above)
Comment (dbkr) [reply sydent/pipeline.yml]: uh, probably just copied from somewhere

File: sydent/pipeline.yml (hunk 2 above)
Comment (dbkr) [reply sydent/pipeline.yml]: mm, good point

**Review by richvdh — APPROVED** [review]
lgtm. Would be nice to see it running somewhere but maybe that's tricky before it lands?

**Comment by dbkr** [comment]
Bleh, I wrote 4 words of a reply on this and then forgot about it for a week. But yes, we could fudge all the branches to get it running and then change them all across the various projects once we merge things - I'm not sure it's that worthwhile in this case.

**Pull request merged**